api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
That means those subpaths are not registered with a webservice and does not have Swagger/OpenAPI spec? From the code (both old and new) I can only see registering `/version`. Where do we register `/version/apiextensions-apiserver` etc?